### ISSUE TITLE
Can't ssh into machine while creating it with salt-cloud using softlayer

### ISSUE THREAD

**Issue by stenius** [saltstack/salt#29471]
So, I am having trouble with the creation of hosts on Softlayer.  It creates the host but it looks like it is unable to contact the host and actually start the installation of the minion and times out after a 900 second counter tries to connect to the machine.

In Softlayer, I can look up the details for a host and ssh into the machine before the salt-cloud step times out and exits.

I am running the latest version of the salt master from git dev branch using salt bootstrap.

Thanks.

`[root@saltm cloud.providers.d]# salt-cloud -p softlayer_1024 www5`
```
[INFO    ] salt-cloud starting
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] Creating Cloud VM www5
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Virtual_Guest
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[INFO    ] POST https://api.softlayer.com/xmlrpc/v3.1/SoftLayer_Account
[ERROR   ] Port connection timed out: 900
Error: There was a profile error: Failed to authenticate against remote ssh

real    16m12.772s
user    0m1.921s
sys     0m0.319s
```

```
softlayer:
    ----------
    softlayer:
        ----------
        www5:
            ----------
            accountId:
                123456
            createDate:
                2015-12-06T17:48:19-06:00
            dedicatedAccountHostOnlyFlag:
                False
            domain:
                example.com
            fullyQualifiedDomainName:
                www5.example.com
            globalIdentifier:
                aaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaa
            hostname:
                www5
            id:
                111111111
            lastPowerStateId:
            lastVerifiedDate:
            maxCpu:
                1
            maxCpuUnits:
                CORE
            maxMemory:
                1024
            metricPollDate:
            modifyDate:
                2015-12-06T17:50:50-06:00
            primaryBackendIpAddress:
                10.76.123.123
            primaryIpAddress:
                184.173.123.123
            provisionDate:
                2015-12-06T17:50:50-06:00
            startCpus:
                1
            status:
                ----------
                keyName:
                    ACTIVE
                name:
                    Active
            statusId:
                1001
            uuid:
                aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa
```
```
softlayer_1024:
  provider: softlayer
  image: UBUNTU_14_64
  location: hou02

  cpu_number: 1
  ram: 1024
  disk_size: 25
  local_disk: False
  hourly_billing: True
  domain: example.com

```
```
[root@saltm salt]# salt --versions-report www2
Salt Version:
           Salt: 2015.8.3-3368-g20ddd4c
 
Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.3.1
         Python: 2.7.5 (default, Jun 24 2015, 00:41:19)
           RAET: Not Installed
        Tornado: 4.1
            ZMQ: 3.2.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed
 
System Versions:
           dist: centos 7.1.1503 Core
        machine: x86_64
        release: 3.10.0-229.el7.x86_64
         system: CentOS Linux 7.1.1503 Core
```

**Comment by ssgward** [comment]
@stenius - thanks, we'll look into this.

**Comment by rallytime** [comment]
I can confirm this bug. However, note that the bug only occurs on the `develop` branch, but does not occurs on the `2015.8` branch or recent releases.

**Comment by stenius** [comment]
I can confirm this has been fixed in salt 2015.8.7 (Beryllium) with SoftLayer==4.1.1
